emperor/format.py

```diff
@@ -54,23 +57,23 @@ def format_pcoa_to_js(header, coords, eigvals, pct_var, custom_axes=[],
 
     Output:
     string: javascript representation of the PCoA data inputed, contains a list
-    of spheres, list of ellipses (if coords_low and coords_high are present) and
-    several setup variables.
+    of spheres, list of ellipses (if coords_low and coords_high are present)
+    and several setup variables.
 
-    Formats the output of qiime.parse.parse_coords_file into javascript variable
-    declarations.
+    Formats the output of qiime.parse.parse_coords_file into javascript
+    variable declarations.
     """
     js_pcoa_string = ''
 
     # validating that the number of coords in coords
-    if number_of_axes>len(coords[0]):
+    if number_of_axes > len(coords[0]):
         number_of_axes = len(coords[0])
 
     # validating that all the axes are above 0.01%, this accounts for really
     # small variations explained in some axes that end up being not practical
     # as the GUI has some problems when presenting those values on screen
-    valid_pcoalabels = len([i for i in pct_var if i>0.01])
-    if number_of_axes>valid_pcoalabels:
+    valid_pcoalabels = len([i for i in pct_var if i > 0.01])
```

Comment:
I'm curious, what is the rationale? List comprehensions are very pythonic and readable, unless the filtering function happens to be eg a builtin.